code/modules/admin/verbs/deathsquad.dm

```diff
@@ -70,26 +70,24 @@ GLOBAL_VAR_INIT(deathsquad_sent, FALSE)
 		return
 
 	// Spawns commandos and equips them.
+	var/list/commando_spawn_locations = list()
 	for(var/obj/effect/landmark/spawner/ds/L in GLOB.landmarks_list) //Despite obj/effect/landmark/spawner/ds being in the exact same location and doing the exact same thing as obj/effect/landmark/spawner/ert, switching them breaks it?
-		if(commando_number == 0)
-			break
-		if(!length(commando_ghosts))
-			break
+		commando_spawn_locations += L
 
-		var/mob/ghost_mob = pick(commando_ghosts)
-		commando_ghosts -= ghost_mob
+	for(var/mob/ghost_mob in commando_ghosts)
 		if(!ghost_mob || !ghost_mob.key || !ghost_mob.client)
 			continue
 
+		var/my_spawn_loc = pick_n_take(commando_spawn_locations)
```

Comment:
this will run into issues as you can spawn more DS than there are spawn points.

Reply:
the maximum amount of commandos able to be sent is now the exact amount of spawn points, instead of a definition, so they wont actually run out